lldb/source/Plugins/Language/Swift/CMakeLists.txt

```diff
@@ -27,6 +27,7 @@ add_lldb_library(lldbPluginSwiftLanguage PLUGIN
     lldbPluginTypeSystemSwift
     swiftAST
     swiftClangImporter
+    swiftCore
```

Comment:
Can you add a comment here that this works around an issue on windows and maybe even ifdef this so we only do this on windows?

Reply:
Fixed 👍